Build fails with clang-60: token is not a valid binary operator in a preprocessor subexpression

```
/usr/ports/math/openturns/work/openturns-1.12/lib/include/openturns/OTdebug.h:52:53: error: token is not a valid binary operator in a preprocessor subexpression
#    if defined(__GNU_LIBRARY__) && ! __GLIBC_PREREQ(2, 10)
                                    ~~~~~~~~~~~~~~~~^
1 error generated.
```

FreeBSD 11.2

--- a/ChangeLog
+++ b/ChangeLog
@@ -41,6 +41,7 @@
  * #1000 (SaltelliSensitivityAlgorithm with LowDiscrepancyExperiment produces wrong results)
  * #1024 (The unary operator is undefined for Normal distribution)
  * #1025 (Empty throw in PythonWrappingFunctions)
+ * #1028 (Build fails with clang-60: token is not a valid binary operator in a preprocessor subexpression)
 
 == 1.12 release (2018-11-08) == #release-1.12
 

--- a/lib/include/openturns/OTdebug.h
+++ b/lib/include/openturns/OTdebug.h
@@ -47,13 +47,6 @@
 #   define REINTERPRET_CAST(T,V) ( reinterpret_cast< T > (V) )
 #  endif /* GCC 3.4.6 test */
 
-#  if !defined(WIN32) && !defined(__APPLE__) /* __GLIBC_PREREQ not defined on mingw or OSX */
-/* The pthread library is bogus for glibc < 2.10 */
-#    if defined(__GNU_LIBRARY__) && ! __GLIBC_PREREQ(2, 10)
-#     define BOGUS_PTHREAD_LIBRARY 1
-#    endif
-#  endif
-
 
 #  ifdef SWIG
 #   define DEPRECATED

